samples/Calling/src/app/views/CallScreen.tsx

```diff
@@ -97,7 +97,7 @@ export const CallScreen = (props: CallScreenProps): JSX.Element => {
       /* @conditional-compile-remove(PSTN-calls) */
       alternateCallerId,
       /* @conditional-compile-remove(rooms) */
-      options: { roleHint }
+      options: roleHint ? { roleHint } : undefined
```

Comment:
this still needs extracted and memoized

```ts
  const callAdapterOptions: AzureCommunicationCallAdapterOptions = useMemo(() => ({
    roleHint
  }), [roleHint]);
```

Reply:
The ternary suffices but sure I'll do that for correctness.

Reply:
doesnt this say if roleHint exists then return a new JSON object which contains the roleHint else undefined? so it would always be new?